docs/general/builtins/scalar-functions.rst

```diff
@@ -4243,6 +4243,64 @@ Special functions
 =================
 
 
+.. _scalar_knn_match:
+
+
+``knn_match(float_vector, float_vector, int)``
+----------------------------------------------
+
+The ``knn_match`` function uses a k-nearest
+neighbour (kNN) search algorithm to find vectors that are similar
+to a query vector.
+
+The first argument is the column to search.
+The second argument is the query vector.
+The third argument is the number of nearest neighbours to search and return.
```

Comment:
I saw the null-handling test, could add our usual description of `NULL` handling to the docs.

Reply:
We usually only document the NULL behavior if it diverges from the default `null -> null`

Reply:
I saw quite some mentioned of NULL handling in sacra-function.rst but also some without, I guess it's not critical